atomic_qt_design/qml/Constants/API.qml

```diff
@@ -43,6 +43,8 @@ QtObject {
 
         enableable_coins: [{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"BTC","name":"Bitcoin","type":"UTXO","explorer_url":"https://blockstream.info/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"LTC","name":"Litecoin","type":"UTXO","explorer_url":"https://blockexplorer.one/litecoin/mainnet/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"DASH","name":"Dash","type":"UTXO","explorer_url":"https://explorer.dash.org/"},{"objectName":"","active":false,"is_claimable":true,"minimal_balance_for_asking_rewards":"10","ticker":"KMD","name":"Komodo","type":"Smart Chain","explorer_url":"https://kmdexplorer.io/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"TUSD","name":"True USD","type":"ERC-20","explorer_url":"https://etherscan.io/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"USDC","name":"USD Coin","type":"ERC-20","explorer_url":"https://etherscan.io/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"CHIPS","name":"Chips","type":"Smart Chain","explorer_url":"https://explorer.chips.cash"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"DEX","name":"Dex","type":"Smart Chain","explorer_url":"https://dex.explorer.dexstats.info/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"ETH","name":"Ethereum","type":"ERC-20","explorer_url":"https://etherscan.io/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"BCH","name":"Bitcoin Cash","type":"UTXO","explorer_url":"https://explorer.bitcoin.com/bch/"},{"objectName":"","active":false,"is_claimable":false,"minimal_balance_for_asking_rewards":"0","ticker":"AWC","name":"Atomic Wallet Coin","type":"ERC-20","explorer_url":"https://etherscan.io/"}],
 
+        get_price_amount: (base_amount, rel_amount) => (parseFloat(rel) / parseFloat(base)).toFixed(8).toString(),
```

Comment:
need to use decimal for arithmetic operations such as divisions

Reply:
This is Mock API, used only in Design Studio.